fieldservice_maintenance/wizard/maintenance_equipment_wizard.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class MainenanceEquipmentWizard(models.TransientModel):
+    _name = "maintenance.equipment.wizard"
+    _description = "Mainenance Equipment Wizard"
+
+    def convert_maintenance_equipment_fsm(self):
+        maintenance_equpment_ids = self.env["maintenance.equipment"].browse(
+            self._context.get("active_ids", [])
+        )
+        for maintenance_id in maintenance_equpment_ids:
+            if not (maintenance_id.product_id or maintenance_id.lot_id):
```

Comment:
Hi there! Thanks for this pr.
I have a problem with this line.
It leads to a bug since there is no product_id on a maintenance.equipment.